src/pudl/transform/ferc1.py

```diff
@@ -5132,6 +5132,60 @@ def process_dbf(self: Self, raw_dbf: pd.DataFrame) -> pd.DataFrame:
         raw_dbf = super().process_dbf(raw_dbf)
         return raw_dbf
 
+    def remove_rare_utility_type_subdimensions_rows(
+        self: Self, df: pd.DataFrame
+    ) -> pd.DataFrame:
+        """Remove the rare, non-total utility types when all values are duplicated.
+
+        This data isn't incorrect, it just interferes with how we process the calculations
+        embedded within these tables.
+
+        """
+        xbrl_factoid = "income_type"
+        dimension_col = "utility_type"
+        idx = ["utility_id_ferc1", "report_year", xbrl_factoid]
+
+        # first we need to find the utility_types where there are mostly
+        # only totals. Which is to say there are a few rare non-total
+        # utility_type's.
+        tot_mask = df[dimension_col] == "total"
+        util_type_count = pd.merge(
+            pd.DataFrame(df.loc[tot_mask, xbrl_factoid].value_counts()),
+            pd.DataFrame(df.loc[~tot_mask, xbrl_factoid].value_counts()),
+            on=xbrl_factoid,
+            how="outer",
+            suffixes=("_total", "_other"),
+        )
```

Comment:
Having trouble visualizing what this is.

Reply:
i added some inline comments explaining